src/main/java/seedu/duke/Profile/Profile.java

```diff
@@ -1,2 +1,48 @@
-package seedu.duke.Profile;public class Profile {
+package seedu.duke.Profile;
+
+public class Profile {
```

Comment:
was thinking if we need to store the name of the user in the future, but looks good for now

Reply:
Same, i not sure if we should be creating multiple profiles for multiple users. potential expansion?